test/parallel/test-stream-readable-unpipe-resume.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const common = require('../common');
+
+const stream = require('stream')
+
+const fs = require('fs');
+const readStream = fs.createReadStream('out/Release/node')
+
+const transformStream = new class extends stream.Transform {
+  _transform() {
```

Comment:
Can you also make sure to wrap this in a `common.mustCall()` to make sure the # of calls is fixed?